CHANGELOG.md

```diff
@@ -47,64 +56,100 @@ and removing some features that no longer seem necessary.
   Legacy versions of Sass (Node & Ruby) are no longer supported
 - BREAKING: The old "init" module, which provided light-weight
   browser normalization, is no longer included. We now recommend
-  using [CSS Remedy](https://github.com/jensimmons/cssremedy).
+  using [CSS Remedy](https://github.com/jensimmons/cssremedy). (_Note: If
+  making the switch from Accoutrement/Init to CSS Remedy, you may notice changes
+  in vertical spacing and the displaying of SVGs._)
 - BREAKING: The old "core" module has been broken into individual modules
-  for "utilities" (extending several Sass modules),
-  "tokens" (the map syntax parser),
-  "variables" (converting Sass variables & maps to CSS custom properties),
-  and "ratios" (for typographic scales and layout aspect ratios)
+  for [utilities][utils] (extending several Sass modules),
+  [tokens][token] (the map syntax parser),
+  [variables][vars] (converting Sass variables & maps to CSS custom properties),
+  and [ratios][ratio] (for typographic scales and layout aspect ratios)
+
+- [Animate][animate]:
+
+  - NEW: Named default easings are now available as [individual variables](animate-ease.html),
+    (without the `_` prefix that is applied in token maps)
 
 - [Sass Utilities][utils]:
 
   - NEW: These functions are now available directly,
     as well as being registered in the map syntax
+  - BREAKING: Private function `_a_list-template()` renamed `template()`
```

Comment:
For functions that are still private, we can leave them out of this documentation entirely. For functions that used to be private, but are now public, we can consider them `new` rather than `breaking`.

Reply:
Removed this and other private functions. Changed the functions that are newly public to `NEW` and added descriptions.